csfieldguide/static/interactives/matrix-simplifier/scss/matrix-simplifier.scss

```diff
@@ -43,15 +43,30 @@
 
 
 #matrix-output,
-#vector-output{
+#vector-output {
   > span {
     display: inline !important;
   }
 }
 
+.close {
+  color: #7f7f7f;
+}
+
+.close:focus, .close:active {
+  outline: none !important;
+  box-shadow: none;
+}
```

Comment:
The x is darker than the pencil when you mouse over them, perhaps try to make them have the same hover colour